### PR TITLE
KAFKA-5738: Upgrade note for cumulative count metric (KIP-187)

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@ijuma Can you review please? Thank you.

File: docs/upgrade.html [hunk 1]
```diff
@@ -74,6 +74,7 @@
         will return <code>0.0</code> in such cases to minimise the probability of breaking users who read the value of every client
         metric (via a <code>MetricsReporter</code> implementation or by calling the <code>metrics()</code> method).
         <code>org.apache.kafka.common.Metric#metricValue()</code> can be used to retrieve numeric and non-numeric metric values.</li>
+    <li>Every Kafka rate metric now has a corresponding cumulative count metric to simplify downstream processing.</li>
```
Comment (ijuma) [inline docs/upgrade.html]: Maybe worth mentioning the typical pattern? That is, `-count` becomes `-total`.

Comment (rajinisivaram) [reply docs/upgrade.html]: @ijuma Thank you for the review, updated.

**Review by ijuma — APPROVED** [review]
LGTM

**Comment by rajinisivaram** [comment]
@ijuma Thank you, merging to trunk.

**Pull request closed without merging**